openstack/networking/v2/ports/results.go

```diff
@@ -54,6 +54,13 @@ type AddressPair struct {
 	MACAddress string `json:"mac_address,omitempty"`
 }
 
+//This is the sub-struct that represents extra DHCP option pairs
```

Comment:
Remember to make sure there is a space after `//` in comments or the comment will not be rendered as documentation in `godoc`.

Also instead of using "This is..." I recommend using the field name: "ExtraDHCPOpts represents extra..."